dev-tools/packer/scripts/Makefile

```diff
@@ -0,0 +1,124 @@
+DATE:=$(shell date +%y%m%d%H%M%S)
+BUILDID?=$(DATE)
+
+makefile_abspath:=$(abspath $(lastword $(MAKEFILE_LIST))/..)
+packer_absdir=$(shell dirname ${makefile_abspath})
+
+
+%/deb: % build/god-linux-386 build/god-linux-amd64 fpm-image
```

Comment:
I split the original Makefile into 2 Makefiles and added $(packer_absdir) to fix path locations